lib/zlog.c

```diff
@@ -521,6 +521,36 @@ void zlog_sigsafe(const char *text, size_t len)
 	}
 }
 
+void _zlog_assert_failed(const struct xref_assert *xref, const char *extra, ...)
+{
+	va_list ap;
+	static bool assert_in_assert = false;
```

Comment:
style nit reported about this init?

Reply:
fixed, sorry about that (and the delay, was AFK-ish last week)